avro-serializer/src/test/java/io/confluent/kafka/serializers/KafkaAvroSerializerTest.java

```diff
@@ -147,14 +153,37 @@ public void testKafkaAvroSerializer() {
     assertEquals("abc", avroDecoder.fromBytes(bytes));
 
     bytes = avroSerializer.serialize(topic, "abc".getBytes());
-    assertArrayEquals("abc".getBytes(), (byte[])avroDeserializer.deserialize(topic, bytes));
-    assertArrayEquals("abc".getBytes(), (byte[])avroDecoder.fromBytes(bytes));
+    assertArrayEquals("abc".getBytes(), (byte[]) avroDeserializer.deserialize(topic, bytes));
+    assertArrayEquals("abc".getBytes(), (byte[]) avroDecoder.fromBytes(bytes));
 
     bytes = avroSerializer.serialize(topic, new Utf8("abc"));
     assertEquals("abc", avroDeserializer.deserialize(topic, bytes));
     assertEquals("abc", avroDecoder.fromBytes(bytes));
   }
 
+  @Test(expected = SerializationException.class)
+  public void testKafkaAvroSerializerWithoutAutoRegister() {
+    avroSerializer.configure(ImmutableMap.of(KafkaAvroDeserializerConfig
+            .SCHEMA_REGISTRY_URL_CONFIG, "bogus", KafkaAvroSerializerConfig.AUTO_REGISTER_SCHEMAS,
```

Comment:
nit: formatting is a bit funky here with a field access split across a newline. can we format this better to fit w/ the style we've been adopting elsewhere, putting args on their own line if they can't all fit in the max line width?